Dockerfile

```diff
@@ -1,20 +1,26 @@
-FROM node:10.15.3-alpine
+ARG PARENT_VERSION=1.0.0-node12.16.0
+ARG PORT_DEBUG=9229
```

Comment:
This ARG doesn't need to be a global arg. It's only used in the development stage, and isn't required by a FROM line, so should just be declared in the stage that it's used. Line 10 should be
`ARG PORT_DEBUG=9229`
and this line should be removed.

Reply:
The lcov.info error was related to a hard coded path in the jenkins library cleanup function.
This has been fixed with the library upgrade.